File: solutions/javascript/raindrops/2/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (x) => {
  let res = "";
  if (x % 3 === 0) res += "Pling";
  if (x % 5 === 0) res += "Plang";
  if (x % 7 === 0) res += "Plong";
  return res || x.toString();
};

File: solutions/python/raindrops/1/raindrops.py
def convert(nu):
    n = ""
    if nu % 3 == 0:
        n += "Pling"
    if nu % 5 == 0:
        n += "Plang"
    if nu % 7 == 0:
        n += "Plong"
    if n == "":
        n = str(nu)
    return n
